src/olympia/users/forms.py

```diff
@@ -395,6 +396,14 @@ def __init__(self, *args, **kwargs):
             if not self.instance.is_developer:
                 choices = email.NOTIFICATIONS_CHOICES_NOT_DEV
 
+            if self.instance.fxa_migrated():
+                self.fields['email'].required = False
+                self.fields['email'].widget = forms.EmailInput(
+                    attrs={'disabled': 'disabled'})
```

Comment:
not `readonly`? [Here](https://kreotekdev.wordpress.com/2007/11/08/disabled-vs-readonly-form-fields/) are the differences. One thing that might be troublesome is the user won't be able to select the text in the email field when it's disabled. I don't know if that's a problem but maybe they would need to copy/paste it?